consensus/src/execution_ctx.rs

```diff
@@ -286,40 +286,30 @@ impl<'a, DB: Database, T: Operations + 'static> ExecutionCtx<'a, DB, T> {
             }
         }
 
-        let ret = phase
+        let msg_topic = msg.topic();
+        let msg_iter = msg.header.iteration;
+        let msg_step = msg.header.get_step();
+        let msg_round = msg.header.round;
+        trace!("collecting msg {msg:#?}");
+
+        let collected = phase
             .lock()
             .await
-            .collect(msg.clone(), &self.round_update, committee)
+            .collect(msg, &self.round_update, committee)
             .await;
 
-        match ret {
-            Ok(output) => {
-                trace!("message collected {:#?}", msg);
```

Comment:
Can we keep this trace or maybe it's redundant?

Reply:
I moved 3lines above